src/main/java/net/shadew/util/misc/ColorUtil.java

```diff
@@ -1370,21 +1370,23 @@ public static float huef(int color) {
         float cmin = MathUtil.min(r, MathUtil.min(g, b));
 
         float delta = cmax - cmin;
-        if (delta == 0) return 0;
```

Comment:
Why did you change this into `return hue` if `hue` is always 0? That's an unnecessary variable allocation :p

Reply:
I just noticed by reading, probably because in the process of trying to understand what it did I basically rewrote it all and then slowly reverted back to the original structure and that might be a remainder from the mess i did.